src/components/BookData.js

```diff
@@ -14,16 +14,45 @@ const BookData = (props) => {
 
   return (
     <div className="book-item">
-      <li>
-        <h2>{title}</h2>
-        <h4>{author}</h4>
-        <button type="button" onClick={removeEvent}>Remove</button>
-      </li>
+      <div className="content">
+        <div className="book-data">
+          <h4 className="book-category">{category}</h4>
+          <h2 className="book-title">{title}</h2>
+          <h6 className="book-author">{author}</h6>
+          <div className="actions">
+            <button className="button-outline" type="button">Comments</button>
+            <div className="v-divider" />
+            <button className="button-outline" type="button" onClick={removeEvent}>Remove</button>
+            <div className="v-divider" />
+            <button className="button-outline" type="button">Edit</button>
+          </div>
+        </div>
+        <div className="progress-container">
+          <div className="circ-progress-container">
+            <div className="circ-progress" />
+          </div>
+          <div className="progress">
+            <p className="percent">72%</p>
+            <p className="completed">Completed</p>
+          </div>
+          <div className="prog-divider" />
+          <div className="chapter-container">
+            <div>
+              <p className="chapter-label">CURRENT CHAPTER</p>
+              <p className="chapter">Introduction</p>
+            </div>
+            <div>
+              <button className="primary-button" type="button">UPDATE PROGRESS</button>
+            </div>
+          </div>
+        </div>
+      </div>
     </div>
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, please notice that you're displaying this component under a `<ul>` element in your `BookList` component, so in order for the HTML to be valid, please make sure that the container of this `BookData` component is a `<li>`. 🤓 🔧

Reply:
<ul> tag removed